config/wpcalypso.json

```diff
@@ -23,6 +23,7 @@
 		"blogger-plan": false,
 		"comments/management/threaded-view": false,
 		"calypsoify/gutenberg": true,
+		"calypsoify/iframe": false,
```

Comment:
So for the E2E tests to see the iframe, we'll need to flip this on I think. cc @Automattic/flowpatrol 

I'll add a commit in a bit

Reply:
Added in 9cca290